nautobot_circuit_maintenance/forms.py

```diff
@@ -119,44 +118,6 @@ class Meta:  # noqa: D106 "Missing docstring in public nested class"
         nullable_fields = ["status", "ack", "description"]
 
 
-class NoteForm(BootstrapMixin, CustomFieldModelFormMixin, RelationshipModelFormMixin):
```

Comment:
We'll need to update `CircuitMaintenanceForm` to add the `NoteModelFormMixin`. We should be able to base that off of `nautobot.apps.forms.NautobotModelForm`